src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -0,0 +1,59 @@
+package com.rtb.projectmanagementtool.auth;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.users.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.io.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String cookieValue;
+  public Cookie currCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "sessionUserID";
+    cookieValue = "out"; // "out" if not logged in
```

Comment:
If this will normally store userIDs (which are ints), I would suggest also storing an int for your "NOT_LOGGED_IN" case.  This way you can easily cast from int<->string without having to worry about if it's a number or not.  You can do zero or -1.